gfauto/gfauto/amber_converter.py

```diff
@@ -260,7 +260,13 @@ def amberscript_uniform_buffer_bind(uniform_json: str, prefix: str) -> str:
     for name, entry in uniforms.items():
         if name.startswith("$"):
             continue
-        if "binding" in entry.keys():
+        if entry["func"] in ["sampler2D", "sampler3D"]:
+            assert "texture" in entry.keys()
+            if entry["texture"] == "DEFAULT":
+                result += f"  BIND BUFFER default_texture AS combined_image_sampler SAMPLER {prefix}_{name} DESCRIPTOR_SET 0 BINDING 0\n"
```

Comment:
I think there should always be a binding field, and we should use that instead of 0.

Reply:
Done